pyproject.toml

```diff
@@ -207,6 +207,9 @@ watchdog = "~2.1.9"
 [tool.poetry.scripts]
 nautobot-server = "nautobot.core.cli:main"
 
+[tool.poetry.group.dev.dependencies]
+mkdocs-python-classy = {git = "https://github.com/itdependsnetworks/mkdocs-python-classy.git", rev = "main"}
```

Comment:
Depending on how Poetry handles this, the biggest blocker is this must be available in PyPI.

1.3 got held up due to Git imports. PyPI doesn't allow it: https://github.com/nautobot/nautobot/pull/1657, https://github.com/nautobot/nautobot/actions/runs/2185417492/jobs/3209161850 (logs gone but this was the failed release).